src/pages/commissions/commission-list.json

```diff
@@ -8,70 +8,75 @@
         "catchPhrase": "Pour vous accompagner dans le voyage",
         "members": [
             {
-                "name": "Tom Demont",
-                "imageURL": "tomdemont.jpg",
-                "websiteURL": "https://people.epfl.ch/tom.demont",
+                "name": "Faourk Boukil",
+                "imageURL": "faroukboukil.jpg",
+                "websiteURL": "https://people.epfl.ch/farouk.boukil",
                 "role": "Président"
             },
             {
-                "name": "Maëlys Billon",
-                "imageURL": "maelysbillon.jpg",
-                "websiteURL": "https://people.epfl.ch/maelys.billon",
+                "name": "Nicolas Wagner",
+                "imageURL": "nicolas.wagner.jpg",
+                "websiteURL": "https://people.epfl.ch/nicolas.wagner",
                 "role": "Vice-présidente"
             },
             {
-                "name": "Alessio Verardo",
-                "imageURL": "../ic-travel.svg",
-                "websiteURL": "https://people.epfl.ch/alessio.verardo",
-                "role": "Administrateur-Trésorier"
+                "name": "Juliette Parchet",
+                "imageURL": "julietteparchet.jpg",
+                "websiteURL": "https://people.epfl.ch/juliette.parchet",
+                "role": "Administratrice-Trésorière"
             },
             {
-                "name": "Eloi Garandel",
+                "name": "Iana Culic",
                 "imageURL": "../ic-travel.svg",
-                "websiteURL": "https://people.epfl.ch/eloi.garandel",
-                "role": "Administrateur-Trésorier"
+                "websiteURL": "https://people.epfl.ch/iana.culic",
+                "role": "Sponsoring"
             },
             {
-                "name": "Louise Rieupouilh",
-                "imageURL": "../ic-travel.svg",
-                "websiteURL": "https://people.epfl.ch/louise.rieupouilh",
-                "role": "Responsable Communication"
+                "name": "Fouad Mahmoud",
+                "imageURL": "fouadmahmoud.jpg",
```

Comment:
```suggestion
                "imageURL": "../ic-travel.svg",
```

Reply:
Error in file
<img width="1377" alt="Capture d’écran 2022-03-06 à 09 55 08" src="https://user-images.githubusercontent.com/43422546/156916068-e5de6ead-dbc7-41f2-b4d5-2401025bca40.png">